web/src/js/__tests__/components/ContentView/ContentLoaderSpec.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import renderer from 'react-test-renderer'
+import ContentLoaderGenerator from '../../../components/ContentView/ContentLoader'
+import { TFlow } from '../../ducks/tutils'
+import TestUtils from 'react-dom/test-utils'
+import mockXMLHttpRequest from 'mock-xmlhttprequest'
+
+global.XMLHttpRequest = mockXMLHttpRequest
```

Comment:
Because we use `XMLHttpRequest` here:
https://github.com/mitmproxy/mitmproxy/blob/5f91b7a45d0b983d3e0f3fff1be8af93736d35c6/web/src/js/components/ContentView/ContentLoader.jsx#L59
Introducing `mock-xmlhttprequest` dependency can save us a lot of troubles.